java-frontend/src/test/files/se/FlowComputation.java

```diff
@@ -48,5 +48,13 @@ void invocation_target(Object a) {
     }
   }
 
+  void test(Object a, Object b) {
+    if (a == b) { // flow@catof
```

Comment:
Don't know why you added this test without the flow message.  To me it already does not have two messages.